pilot/pkg/networking/util/util.go

```diff
@@ -257,15 +257,21 @@ func SortVirtualHosts(hosts []route.VirtualHost) {
 	})
 }
 
-// IsProxyVersionGE11 checks whether the given Proxy version is greater than or equals 1.1.
-func IsProxyVersionGE11(node *model.Proxy) bool {
-	ver, _ := node.GetProxyVersion()
-	return ver >= "1.1"
+// IsIstioVersionGE12 checks whether the given Istio version is greater than or equals 1.2.
+func IsIstioVersionGE12(node *model.Proxy) bool {
+	ver, _ := node.GetIstioVersion()
+	return ver >= "1.2"
+}
+
+// IsIstioVersionGE13 checks whether the given Istio version is greater than or equals 1.3.
+func IsIstioVersionGE13(node *model.Proxy) bool {
+	ver, _ := node.GetIstioVersion()
+	return ver >= "1.3"
```

Comment:
This check and the existing GE11 are not safe. Consider if we release a patch to all versions. Then you can have a feature in Istio 1.12 that is not supported by Istio 1.2.2.

I guess if we have something like this then we can update pilot at the same time though

Reply:
yeah. that was the thinking